graph/cycle_detection.py

```diff
@@ -44,7 +44,7 @@ def contains_cycle(graph):
     traversal_states = {vertex: TraversalState.WHITE for vertex in graph}
     for vertex, state in traversal_states.items():
         if (state == TraversalState.WHITE and
-            is_in_cycle(graph, traversal_states, vertex)):
+           is_in_cycle(graph, traversal_states, vertex)):
```

Comment:
@keon Added this commit to avoid the following pep8 error, which i was getting while running the style checks:
**visually indented line with same indent as next logical line**

As per pep8,
When the conditional part of an if -statement is long enough to require that it be written across multiple lines, it's worth noting that the combination of a two character keyword (i.e. if ), plus a single space, plus an opening parenthesis creates a natural 4-space indent for the subsequent lines of the multiline conditional. This can produce a visual conflict with the indented suite of code nested inside the if -statement, which would also naturally be indented to 4 spaces.